### ISSUE TITLE
code 

### ISSUE THREAD

**Issue by qizhuang-qz** [weiyikang/FedGM_torch#1]
Hello, when will the main_dg. py code be released

**Comment by weiyikang** [comment]
About June. I also focus on your work FedCSPC.

**Comment by qizhuang-qz** [comment]
hello，I would like to compare your method as the SOTA approach in my recent work. Would it be possible for you to provide the training code, especially the loss function?

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by weiyikang** [comment]
the train code, include intra_gm and inter_gm.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "weiyikang/FedGM_torch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月23日(星期六) 中午12:41
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [weiyikang/FedGM_torch] code  (Issue #1)





 hello，I would like to compare your method as the SOTA approach in my recent work. Would it be possible for you to provide the training code, especially the loss function? 
 
 获取Outlook for Android<https://aka.ms/AAb9ysg&gt; 
 ________________________________
